frontend/packages/dev-console/src/components/edit-application/edit-application-types.ts

```diff
@@ -0,0 +1,24 @@
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { FirehoseResult } from '@console/internal/components/utils';
+import { NormalizedBuilderImages } from '../../utils/imagestream-utils';
+
+export interface EditApplicationFormProps {
+  builderImages?: NormalizedBuilderImages;
+}
+
+export interface AppResources {
+  service?: FirehoseResult;
+  route?: FirehoseResult;
+  buildConfig?: FirehoseResult;
+  imageStreams?: FirehoseResult;
+}
+
+export interface EditApplicationProps {
```

Comment:
These props should be co-located with the component. I don't see them being used anywhere else.

You can also pull prop types out of a component for re-use by doing:
`React.ComponentProps<typeof EditApplication>`

Reply:
This is being used at 2 places one in loader component and other in the formik component,